### ISSUE TITLE
How to set input range?

### ISSUE THREAD

**Issue by DanielArnett** [resolume/ffgl#15]
How do we set the range on an input? By default all of the inputs on the example are from 0 to 1. How do I switch them to a different range?

**Comment by ResolumeMenno** [comment]
You can't.
ffgl FF_TYPE_STANDARD parameters always have to be in the range between 0.0f and 1.0f. However, since you are storing the values yourself in the plugin you can remap that range to whatever range you may need. You can then use the parameter display value to tell the host how to show the value of that parameter.

I've updated the particles example for you to show how to override the GetParameterDisplay function.
In short you do this:
1. In SetFloatParameter you convert the value from 0.0f..1.0f into your custom range.
2. In GetFloatParameter you convert your value from your custom range into the 0.0f..1.0f range.
3. In GetParameterDisplay you output the textual representation of the current value.

**Comment by DanielArnett** [comment]
Okay I guess that's good enough for my needs, thank you.

**Comment by DanielArnett** [comment]
Hmm, I've switched to the ffgl1.5 branch and have started migrating my shaders (trying to make them compatible with more versions of ffgl). But when I try to change the parameter names, or remove parameters I still get this.
![image](https://user-images.githubusercontent.com/3029502/48144175-6569f680-e27e-11e8-9fab-133e1e183a79.png)
It's based off  the AddSubtract Example, and even though I comment out some of the FFPARAMs, they still show up in resolume. Notice though that changes like the effect name are working though.

**Comment by DanielArnett** [comment]
Ahh, got it. Switched the name of my plugin and didn't delete the old .dll file from my binaries folder, so I had two copies in the same folder and my changes weren't updating.
